api/lib/infrastructure/repositories/sco-certification-candidate-repository.js

```diff
@@ -18,7 +19,15 @@ const addNonEnrolledCandidatesToSession = async function ({ sessionId, scoCertif
     .filter((candidate) => !alreadyEnrolledCandidateOrganizationLearnerIds.includes(candidate.organizationLearnerId))
     .map(scoCandidateToDTO);
 
-  await knex.batchInsert('certification-candidates', candidatesToBeEnrolledDTOs);
+  const addedCandidateIds = await knex
+    .batchInsert('certification-candidates', candidatesToBeEnrolledDTOs)
+    .returning('id as certificationCandidateId');
+
+  addedCandidateIds.forEach((candidate) => {
+    candidate.type = SubscriptionTypes.CORE;
+  });
```

Comment:
J'ai pas reussi sans `knex.raw` mais on peut eviter le map
```suggestion
  const addedCandidateIds = await knex
    .batchInsert('certification-candidates', candidatesToBeEnrolledDTOs)
    .returning(knex.raw(`id as "certificationCandidateId", '${SubscriptionTypes.CORE}' as type`));
```

Reply:
Classe !

Reply:
Done